Sources/Serveable.swift

```diff
@@ -0,0 +1,10 @@
+
+/// Serves your order to a local or a remote bar
+
+public protocol Serveable {
+
+    associatedtype S
+
+    func serve(order: Order, delivery: (Delivery<S>)->())
```

Comment:
I'm no fan of general words like handler / manager that have no meaning or added value because they contain no information about the variable. In the original syntax we would call this `result`. I prefer `result over `completionHandler` but delivery is a synonym and sound more like it would be used in a bar 🤔